docs/approach.rst

```diff
@@ -49,20 +54,25 @@ have good signal for some echoes.
 In order to avoid using bad signal from affected echoes in calculating
 :math:`T_{2}^*` and :math:`S_{0}` for a given voxel, ``tedana`` generates an
 adaptive mask, where the value for each voxel is the number of echoes with
-"good" signal.
-When :math:`T_{2}^*` and :math:`S_{0}` are calculated below, each voxel's values
-are only calculated from the first :math:`n` echoes, where :math:`n` is the
-value for that voxel in the adaptive mask.
+"good" signal. The voxel in the shortest echo with the 33rd percentile mean signal
+across time is identified. The threshold for each echo is the signal in the same voxel
+divided by 3. This is an arbitrary, but conservative threshold in that it only
```

Comment:
Is it always divided by 3, or is it N_Echoes? I can also just check adaptive masking...